site/templates/application/footer/social-media-icons.jet

```diff
@@ -0,0 +1,96 @@
+{{ block footerSocialMediaIcons() }}
```

Comment:
This appears to be unused... and the contents of the file looks like it's just a duplicate of social-media-buttons.jet... can this file be deleted?